server/config/tenant/config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This doesn't contain all of the profile - upon looking at it - I'm not sure that workstack columns (and a few others) should be part of this...

Reply:
There is a ticket to have workstack columns in hocs-casework, so I guess I'm right.